specs/test_formats/genesis/initialization.md

```diff
@@ -4,15 +4,39 @@ Tests the initialization of a genesis state based on Eth1 data.
 
 ## Test case format
 
+### `eth1_block_hash.yaml`
+
+A `Bytes32` hex encoded, with prefix 0x. The root of the Eth-1 block.
+
+A `eth1_block_hash.ssz` is available as substitute.
```

Comment:
Prefer "Also available as" which is used for some of `.ssz` file notes
let's be consistent

Reply:
Fixed, see 18fc4ed